test/workflows/test_meta/test_meta.cwl

```diff
@@ -0,0 +1,23 @@
+cwlVersion: v1.2
+# What type of CWL process we have in this document: ComandLineTool or Workflow.
+class: CommandLineTool
+
+# The inputs for this process: none.
+inputs: []
+# The outputs for this process: none.
+outputs: []
+
+baseCommand: ["echo", "Hello World"]
+
+$namespaces:
+  dirac: "./schemas/dirac-metadata.yaml#" #we place a copy locally, but it should be published
```

Comment:
Typically `$namespaces` should be used to expand a prefix like `direct` into a public URI

You can use `$schemas` if you want to link to the exact schema (in RDF-XML format) that was used. 

https://www.commonwl.org/v1.2/SchemaSalad.html#Explicit_context

Reply:
Thanks a lot for the feedback!

Reply:
@mr-c thank you for the suggestion! I want to shadow our DIRAC-specific hints, and I would like to have native schema validation to check whether hint is having a correct form. Is the use of RDF schema the only way to accomplish this?

Reply:
> Is the use of RDF schema the only way to accomplish this?

https://www.commonwl.org/v1.2/SchemaSalad.html#Explicit_context

RDF-XML, yes